modules/katamari/src/test/ts/browser/RepeatableTest.ts

```diff
@@ -0,0 +1,38 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
+
+import { Optional } from 'ephox/katamari/api/Optional';
+import * as Singleton from 'ephox/katamari/api/Singleton';
+
+const delay = (ms: number): Promise<never> => {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+};
+
+describe('browser.katamari.RepeatableTests', () => {
+  it('Make a repeatable then clean it', async () => {
+    const intervalId = Singleton.repeatable(100);
+    assert.strictEqual(intervalId.get(), Optional.none());
+    let counter = 0;
+    intervalId.set(() => {
+      counter++;
+    });
+    await delay(250);
```

Comment:
I think this could be a waiter that check two things that the counter is greater that 2 and that the now - start time is more than 200ms that way it would be a bit more robust if the browser would lag and fire way the interval call 50ms to late. These time based tests tend to flake, less now I guess when we don't have to deal with IE anymore but still.

Reply:
I added assertions around time. Is that what you meant ?